content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>I would add to that a suggestion about conda, since it's become the standard for most scientific computing applications in Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done.

```diff
This tutorial can be run locally in an isolated environment, such as
[Virtualenv](https://virtualenv.pypa.io/en/stable/)
+ and [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)
...
```

Reply:
<p>nit: I'd say virtualenv <em>or</em> conda, but that's just the mathematician in me talking :)</p>

Reply:
Sounds good @melissawm